### PR TITLE
mostly Gabe's code, merged into the bh script

### PR DESCRIPTION
**Pull request by ZeroChaos-**

### CODE REVIEW

**Comment by granolocks** [comment]
LGTM

**Comment by tmossey** [comment]
lgtm at first pass but has this been tested on device?

**Comment by ZeroChaos-** [comment]
my main man Dallas is testing it, since my pad has bad usb :-(

File: scripts/blue_hydra.sh [hunk 1]
```diff
@@ -7,14 +7,37 @@ clear
...
+       le_mode AS le, le_address_type, 
+       updated_at as last_seen,
+       classic_major_class, classic_minor_class, classic_class 
+FROM blue_hydra_devices 
+WHERE CAST(strftime('%s',updated_at) AS integer) 
+BETWEEN CAST($START_TIME AS integer) AND CAST($STOP_TIME AS integer);
+EOF
+)
+
+echo $QUERY | sqlite3 -header -column /opt/pwnix/blue_hydra.db > $FILENAME
```
Comment (granolocks) [inline scripts/blue_hydra.sh]: i really wnat this line to be indented...

**Comment by bclune** [comment]
`export STOP_TIME="now() AS integer); DROP TABLE blue_hydra_devices; --"`

**Comment by granolocks** [comment]
i would totally care about that if they couldn't already run `rm blue_hydra.db`

**Comment by bclune** [comment]
Heh it's obviously not an issue since whoever is running the script has the permissions to pipe arbitrary queries into `sqlite3`, it's just kind of funny.

**Comment by granolocks** [comment]
haha yea

**Comment by ZeroChaos-** [comment]
Dallas tested successfully so here we go.

**Pull request merged**